code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -637,254 +774,390 @@
+	name = "Mousetrap Mortar"
 	id = "mech_mousetrap_mortar"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/launcher/mousetrap_mortar
 	materials = list(/datum/material/iron=20000,/datum/material/bananium=5000)
 	construction_time = 300
-	category = list(RND_CATEGORY_EXOSUIT_EQUIPMENT)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT,
+		RND_CATEGORY_MECHFAB_HONK
+	)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_HONK,
+		RND_CATEGORY_MECHFAB_HONK + RND_SUBCATEGORY_MECHFAB_SUPPORTED_EQUIPMENT
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
 
 /datum/design/mech_banana_mortar
-	name = "H.O.N.K Weapon (Banana Mortar)"
+	name = "Banana Mortar"
 	id = "mech_banana_mortar"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/launcher/banana_mortar
 	materials = list(/datum/material/iron=20000,/datum/material/bananium=5000)
 	construction_time = 300
-	category = list(RND_CATEGORY_EXOSUIT_EQUIPMENT)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT,
+		RND_CATEGORY_MECHFAB_HONK
+	)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_HONK,
+		RND_CATEGORY_MECHFAB_HONK + RND_SUBCATEGORY_MECHFAB_SUPPORTED_EQUIPMENT
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
 
 /datum/design/mech_honker
-	name = "H.O.N.K Weapon (HoNkER BlAsT 5000)"
+	name = "HoNkER BlAsT 5000"
 	id = "mech_honker"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/honker
 	materials = list(/datum/material/iron=20000,/datum/material/bananium=10000)
 	construction_time = 500
-	category = list(RND_CATEGORY_EXOSUIT_EQUIPMENT)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT,
+		RND_CATEGORY_MECHFAB_HONK
+	)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_HONK,
+		RND_CATEGORY_MECHFAB_HONK + RND_SUBCATEGORY_MECHFAB_SUPPORTED_EQUIPMENT
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
 
 /datum/design/mech_punching_glove
-	name = "H.O.N.K Weapon (Oingo Boingo Punch-face)"
+	name = "Oingo Boingo Punch-face"
 	id = "mech_punching_face"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/launcher/punching_glove
 	materials = list(/datum/material/iron=20000,/datum/material/bananium=7500)
 	construction_time = 400
-	category = list(RND_CATEGORY_EXOSUIT_EQUIPMENT)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT,
+		RND_CATEGORY_MECHFAB_HONK
+	)
+	category = list(
+		RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_HONK,
+		RND_CATEGORY_MECHFAB_HONK + RND_SUBCATEGORY_MECHFAB_SUPPORTED_EQUIPMENT
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
 
 /////////////////////////////////////////
 //////////////Borg Upgrades//////////////
 /////////////////////////////////////////
 
 /datum/design/borg_upgrade_rename
-	name = "Cyborg Upgrade (Rename Board)"
```

Comment:
we are we removing Cyborg Upgrade from the names here?

Reply:
They're only visible in "cyborg modules," so it's redundant. The printed item still has the old name. I also did this for all computer and machine boards (machine design and computer design labels are _dead_)